db/migrate/20200318135632_rename_report_templates.rb

```diff
@@ -0,0 +1,30 @@
+class RenameReportTemplates < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Ansible Inventory" => "Ansible - Ansible Inventory",
+    "Applicable errata" => "Host - Applicable Errata",
+    "Applied Errata" => "Host - Applied Errata",
+    "Entitlements" => "Subscription - Entitlement Report",
+    "Host statuses" => "Host - Statuses",
+    "Registered hosts" => "Host - Registered Content Hosts",
+    "Registered users" => "User - Registered Users",
+    "Subscriptions" => "Subscription - General Report",
+  }
+
+  class FakeReportTemplate < ApplicationRecord
+    self.table_name = 'templates'
+  end
+
+  def up
+    TEMPLATE_NAMES.each do |from, to|
+      FakeReportTemplate.find_by_name(to).delete
+      FakeReportTemplate.find_by_name(from).try(:update_attributes, :name => to)
```

Comment:
To follow new Ruby/Rails conventions:
```suggestion
      FakeReportTemplate.find_by(:name => from)&.update(:name => to)
```

Reply:
Hmm can do partially, I want `update_attributes` to prevent triggering orchestration tho.

Reply:
In that case, you want `update_columns`. `update_attributes` is alias for `update` and it is deprecated in Rails 6 :)

Reply:
or this would work as well:
```suggestion
      FakeReportTemplate.find_by(:name => from)&.update_column(:name, to)
```

Reply:
btw @lzap what orchestration would be triggered? Template dosn't have any orchestration on it, right?

Reply:
if we use `FakeModel`, there's no callback defined, no validation, no orchestration, it's safe to use update, however update_column would be probably slightly faster

Reply:
There is only difference: The `updated_at` won't be updated in case of `update_column(s)`. But I don't care which one we choose :)